files/build_templates/init_cfg.json.j2

```diff
@@ -110,7 +110,9 @@
     "SYSLOG_CONFIG": {
         "GLOBAL": {
             "rate_limit_interval" : "0",
-            "rate_limit_burst" : "0"
+            "rate_limit_burst" : "0",
```

Comment:
>rate_limit_burst [](http://example.com/codeflow?start=13&length=16)

For each GLOBAL attribute, could you define default values in yang model, then you can completely remove SYSLOG_CONFIG table in this file.

Reply:
We are not generating defaults from the YANG model, so that is why we place it here. It cannot be removed

Reply:
@qiluo-msft I think this is a generic comment which needs to be addressed for all fields (and may be for all tables in init_cfg.j2). I would suggest lets open a task and address this change independently rather than being part of this PR.

Reply:
I did not mean a generic comment. I only ask to stop adding new fields into this file:
            "format": "standard",
            "severity": "notice"

The quickest solution is to add hard-coded default value into rsyslog.config.j2, like default port which already in that file.

Reply:
> The quickest solution is to add hard-coded default value into rsyslog.config.j2, like default port which already in that file.

@qiluo-msft Done